gatt_blacklist.txt

```diff
@@ -8,6 +8,13 @@
 # become keyloggers.
 00001812-0000-1000-8000-00805f9b34fb
 
+# Firmware update services that don't check the update's signature
+# present too much of a phishing risk to expose to the open web.
```

Comment:
# present a risk of devices' software being modified by any party.
# Users may connect to a device believing they are enabling
# only simple interaction but resulting in the device being
# compromised.
# 

Perhaps space out the intro description from the list of items? e.g. with that trailing "#" line.